src/components/editor/EditorCanvas.styles.tsx

```diff
@@ -8,17 +8,14 @@ const useStyles = makeStyles()((theme: Theme) => {
                 display: 'flex',
             },
             konvaContainer: {
-                marginTop: theme.spacing(1),
-                height: `100%`,
+                height: 'calc(100vh - 64px)',
```

Comment:
Cannot judge this but shouldn't we avoid usage of `px` ?

Reply:
It is usually better, but in this specific case the `64px` is the height of the top nav bar, which IMO should be in px, as if we use a relative unit, we can get some strange behavior such as this one (caused by the developer console):

![image](https://github.com/kbss-cvut/fta-fmea-ui/assets/57750426/08053821-7291-4788-8c3a-ac185e2a5120)